FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -634,29 +949,37 @@ TickType_t uxIdentifier = 0U;
 
 		#if( ipconfigDNS_USE_CALLBACKS == 1 )
 		{
-			if( pCallback != NULL )
+			if( pCallbackFunction != NULL )
 			{
 				if( ulIPAddress == 0UL )
 				{
 					/* The user has provided a callback function, so do not block on recvfrom() */
 					if( xHasRandom != pdFALSE )
 					{
-						uxReadTimeOut_ticks = 0U;
-						vDNSSetCallBack( pcHostName, pvSearchID, pCallback, uxTimeout, uxIdentifier );
+						uxReadTimeOut_ticks = 0;
+						#if( ipconfigUSE_IPv6 != 0 )
+						{
+							vDNSSetCallBack( pcHostName, pvSearchID, pCallbackFunction, uxTimeout, ( TickType_t )uxIdentifier, ( xFamily == FREERTOS_AF_INET6 ) ? pdTRUE : pdFALSE );
+						}
+						#else
+						{
+							vDNSSetCallBack( pcHostName, pvSearchID, pCallbackFunction, uxTimeout, ( TickType_t )uxIdentifier );
+						}
+						#endif
```

Comment:
Can we add a comment here? Like so: `#endif     /*  ( ipconfigUSE_IPv6 != 0 )  */ `?